pallets/settlement/src/lib.rs

```diff
@@ -1344,7 +1349,7 @@ impl<T: Trait> Module<T> {
                     )
                     .ok();
                 }
-                LegStatus::PendingTokenLock => {}
+                _ => {}
```

Comment:
It's better to explicitly define behavior for all enum variants so that we don't miss any logic.

The idea is that if we explicitly define variants, we'll need to see their names and that might remind us of any logic we need to add to handle them. If we use `_` and add a new variant later, we might forget to handle any logic for that variant in this match.

It's a very nit picky thing but ¯\_(ツ)_/¯

Reply:
Fixed